### ISSUE TITLE
[Bug]: Talkback Audio does not work fine

### ISSUE THREAD

**Issue by kennethbe** [homebridge-eufy-security/plugin#406]
### What happened?

Hello 

so this is a bug that was fixed in [2.2.0-beta.14]
its about talkback function not working and the fix was adding a selector to switch from mono to stereo

this was the original post 

Bug number 196

What happened?

Hey there,

I use a T8200 Video doorbell and want to use talkback Audio.

The problem is, that there is just some fancy noise when I want to speak to the doorbell. Hearing is fine so far.
I'm using a Rapsi4, thats why I only can use less resolution.

### Device Type

Doorbell (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.6.x (Supported)

### NodeJS Version

v18 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
….
```

### config

```shell
"platform": "EufySecurity",
      "username": "mail",
      "password": "pwd",
      "pollingIntervalMinutes": 30,
      "hkHome": 1,
      "hkAway": 0,
      "hkNight": 3,
      "hkOff": 63,
      "enableDetailedLogging": 0,
      "country": "DE",
      "CameraMaxLivestreamDuration": 600,
      "cameras": [
        {
          "serialNumber": "ID",
          "enableCamera": "true",
          "enableButton": false,
          "motionButton": false,
          "unbridge": true,
          "forcerefreshsnap": true,
          "videoConfig": {
            "audio": true,
            "audioSampleRate": 0,
            "useSeparateProcesses": true,
            "maxStreams": 1,
            "maxWidth": 640,
            "maxHeight": 480,
            "maxBitrate": 100,
            "maxFPS": 15
          },
          "talkback": true
        }
```

**Comment by lenoxys** [comment]
Hello,

Yeah I didn't back ported all the stuff from beta because I didn't write the code. So I've to pay attention on this.

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
